### DISCUSSION TITLE
SDF Version number

### DISCUSSION THREAD

**Discussion by clydeugene** [vsbuffalo/scidataflow#26] [Q&A] [answered]
Hi @vsbuffalo 

Is there a way of determining SciDataFlow's version number when citing the tool? 

I don't see the version installed when I run `sdf --help` and it appears the tool does not have `-v` or `--version` flags.

Clyde

**Comment by apraga** [comment, marked as answer]
Hi,
I've made a PR to fix that. In the meantime, you can check Cargo.toml (v0.8.11) and https://github.com/vsbuffalo/scidataflow/blob/main/CITATION.cff (version will be updated in the same PR).
